spring-boot

TheAtlac opened the pull request.

TheAtlac committed:
Выполнено 1 задание

TheAtlac committed:
Выполнено 2 и 3 задания

TheAtlac committed:
добавил test.yml

rassafel reviewed (commented):
Ошибки в валидации:
- Возможно создать/обновить песню с пустым названием песни/артиста
- Возможно прослушать 0 раз песню

TheAtlac committed:
добавлено тестирование
исправлены ошибки с создание песни с пустым именем или артистом
исправлена ошибка с прослушивание 0 раз
